src/validation.h

```diff
@@ -799,7 +799,8 @@ class ChainstateManager
     std::vector<CChainState*> GetAll();
 
     //! The most-work chain.
-    CChain& ActiveChain() const;
+    CChainState& ActiveChainstate() const;
+    CChain& ActiveChain() const { return ActiveChainstate().m_chain; }
```

Comment:
If `ActiveChain()` definition is placed in header, why `ActiveChainstate()` definition is not placed here too?

Reply:
To minimize the diff and aid review